### PR TITLE
Run integration tests.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: package.json [hunk 1]
```diff
@@ -96,18 +96,18 @@
...
     "lint": "./node_modules/.bin/eslint '**/*.js' --quiet",
-    "lint-plugins": "PLUGINS=$npm_config_externalplugins; ./node_modules/.bin/eslint \"$PLUGINS/**/*.js\" --quiet",
+    "lint-plugins": "PLUGINS=$npm_config_externalplugins ./node_modules/.bin/eslint \"$PLUGINS/**/*.js\" --quiet",
     "prebuild": "./scripts/validate-tests",
-    "serve": "NODE_ENV='development' npm run build-assets && ./node_modules/.bin/gulp serve",
+    "serve": "npm run build-assets && ./node_modules/.bin/gulp serve",
     "serve-notify": "export NOTIFY=true && npm run serve",
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
-    "start": "PORT=$npm_package_config_port; NODE_ENV='development' webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --port $PORT",
+    "start": "NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --port $npm_package_config_port",
```
Comment (mlunoe) [inline package.json]: I think we need an entry in the README to do the `npm config set port 4200`

File: webpack/webpack.dev.babel.js [hunk 2]
```diff
@@ -8,37 +8,51 @@ import webpackConfig from './webpack.config.babel';
 
 // Defaults to value in package.json.
 // Can override with npm config set dcos-ui:port 80
```
Comment (mlunoe) [inline webpack/webpack.dev.babel.js]: Shouldn't this be `4200`?

Comment (mlunoe) [inline webpack/webpack.dev.babel.js]: Also I think @MatApple had some issues with the `dcos-ui:` prefix

File: webpack/webpack.dev.babel.js [hunk 3]
```diff
@@ -8,37 +8,51 @@ import webpackConfig from './webpack.config.babel';
...
+let entry = [
+  `webpack-dev-server/client?http://localhost:${PORT}`,
+  './src/js/index.js'
+];
+
+if (environment === 'development') {
+  entry.push('webpack/hot/only-dev-server');
+  devtool = '#eval-source-map';
+} else if (environment === 'testing') {
+  let delayTime = 1000 * 60 * 60 * 5; // 5 hours? seems legit
```
Comment (mlunoe) [inline webpack/webpack.dev.babel.js]: What is going on here? Why is this necessary. Comment is not explanatory

File: package.json (hunk 1 above)
Comment (rcorral) [inline package.json]: We don't need to do that, the default is `4200`

File: webpack/webpack.dev.babel.js (hunk 3 above)
Comment (rcorral) [inline webpack/webpack.dev.babel.js]: Making me rewrite my perfectly good comment :p

Comment (mlunoe) [inline webpack/webpack.dev.babel.js]: Hehe, yes :flushed:

Comment (rcorral) [inline webpack/webpack.dev.babel.js]: Done.

File: package.json (hunk 1 above)
Comment (mlunoe) [inline package.json]: Arh! There it was: https://github.com/dcos/dcos-ui/pull/121/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23

Cool, no problem then

File: webpack/webpack.dev.babel.js (hunk 2 above)
Comment (MatApple) [inline webpack/webpack.dev.babel.js]: This looks good to me. I'm going to migrate our `externalplugins` config back to using `dcos-ui:` prefix but have to find some time one evening to investigate the options in CI. Don't think it will be a problem. Lets stick to using `dcos-ui:` prefix for the port.
